ray/README.md

```diff
@@ -10,14 +10,98 @@ Follow the instructions below to install and configure this check for an Agent r
 
 ### Installation
 
-The Ray check is included in the [Datadog Agent][2] package.
-No additional installation is needed on your server.
+Starting from Agent release 7.49.0, the Ray check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.
+
+<div class="alert alert-warning">This check uses <a href="https://docs.datadoghq.com/integrations/openmetrics/">OpenMetrics</a> to collect metrics from the OpenMetrics endpoint Ray can expose, which requires Python 3.</div>
```

Comment:
```suggestion
**WARNING**: This check uses [OpenMetrics](https://docs.datadoghq.com/integrations/openmetrics/) to collect metrics from the OpenMetrics endpoint Ray can expose, which requires Python 3.
```
This is a fairly new discovery on our side, but these alert boxes are stripped out of the app tiles, so it's best just to use standard markdown to make sure they're visible there.

Reply:
✅